<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGdalSourceSelectBase</class>
 <widget class="QDialog" name="QgsGdalSourceSelectBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Raster Layer(s)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="srcGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Source Type</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout1">
        <item>
         <widget class="QRadioButton" name="radioSrcFile">
          <property name="text">
           <string>F&amp;ile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioSrcProtocol">
          <property name="text">
           <string>Protoco&amp;l: HTTP(S), cloud, etc.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioSrcOgcApi">
          <property name="text">
           <string>OGC API</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="fileGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Source</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="rasterDatasetLabel">
          <property name="text">
           <string>Raster dataset(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsFileWidget" name="mFileWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="protocolGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Protocol</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_21">
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbProtocolTypes"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelProtocolURI">
          <property name="text">
           <string>&amp;URI</string>
          </property>
          <property name="buddy">
           <cstring>protocolURI</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="protocolURI"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelBucket">
          <property name="text">
           <string>Bucket or container</string>
          </property>
          <property name="buddy">
           <cstring>mBucket</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="mBucket"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelKey">
          <property name="text">
           <string>Object key</string>
          </property>
          <property name="buddy">
           <cstring>mKey</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="mKey"/>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QGroupBox" name="mAuthGroupBox">
          <property name="title">
           <string>Authentication</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QgsAuthSettingsWidget" name="mAuthSettingsProtocol" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QgsCollapsibleGroupBox" name="mCredentialOptionsGroupBox" native="true">
       <property name="title" stdset="0">
        <string>Credential Options</string>
       </property>
       <layout class="QVBoxLayout" name="openOptionsVBoxLayout_2" stretch="1">
        <item>
         <layout class="QFormLayout" name="mCredentialOptionsLayout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QgsScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>876</width>
          <height>246</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QgsCollapsibleGroupBox" name="mOpenOptionsGroupBox" native="true">
           <property name="title" stdset="0">
            <string>Options</string>
           </property>
           <layout class="QVBoxLayout" name="openOptionsVBoxLayout">
            <item>
             <widget class="QLabel" name="mOpenOptionsLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="mOpenOptionsLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsAuthSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qgsauthsettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QWidget</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qgsscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mOpenOptionsGroupBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsGdalSourceSelectBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
